source/javascripts/components/unified-editor/WorkflowCard/components/StepCard.tsx

```diff
@@ -86,18 +92,23 @@ const StepCard = ({
   stepBundleId,
 }: StepCardProps) => {
   const zoom = useReactFlowZoom();
+  const enableStepBundles = useFeatureFlag('enable-wfe-step-bundles-ui');
+  const [isToggletipDismissedGlobally, setToggletipDismissedGlobally] = useLocalStorage('toggletipDismissed', false);
```

Comment:
```suggestion
  const [isToggletipDismissedGlobally, setToggletipDismissedGlobally] = useLocalStorage('multiSelectionAccepted', false);
```
Or something like that